### ISSUE TITLE
this needs more stuff!

### ISSUE THREAD

**Issue by luisrudge** [owin-middleware/Registry#18]
hey. Awesome stuff. I was just thinking we REALLY need this to take off. Everyone is building Middlewares, I'm sure. But not everyone is finding those. We need to find a way to make this project more visible. Any thoughts?

**Comment by howarddierking** [comment]
I'm sure there are a handful of things (e.g. adding linked data for optimizing search engines, etc.) - but I suspect that a lot of it is just more frequent references from blogs, etc.

**Comment by luisrudge** [comment]
Yeah. I think we need to reach out to those people and ask them to post things here. Start building a community and then see what we can do to expose data in the best way.

**Comment by ainglese** [comment]
a bit late, but we really need this to take off.
i'm proposing this to be added.. https://github.com/etishor/Metrics.NET/wiki/OWIN-Metrics-Adapter
https://github.com/aspnet/WebHooks
https://github.com/stefanprodan/WebApiThrottle
https://github.com/daprlabs/Orleans.Owin
https://github.com/NLog/NLog.Owin.Logging
https://github.com/Thorium/Owin.Compression
https://github.com/msarchet/Bundler
(absolutely not my stuff)

**Comment by howarddierking** [comment]
@luisrudge would you mind putting this int the form of a pull request?
